spec/unit/provider/mount/windows_spec.rb

```diff
@@ -106,9 +106,17 @@ class Volume
         @provider.mount_fs
       end
 
-      it "should not mount the filesystem if it is mounted" do
-        expect(@vol).not_to receive(:add)
-        allow(@current_resource).to receive(:mounted).and_return(true)
+      it "should remount the filesystem if it is mounted and the options have changed" do
+        @vol.should_receive(:add).with(:remote => @new_resource.device,
+                                       :username => @new_resource.username,
```

Comment:
@JamesBelchamber 
To make travis happy could you please replace @vol.should_receive with following
expect(@vol)..to receive(:add).with(....). As `should_receive` is deprecated in latest rspec version.